autobattlefrontier.js

```diff
@@ -45,11 +52,21 @@ function initBattleFrontier() {
     }
 
     function createHTML() {
+        if (bfOneClickState == "OFF") {
+            bfOneClickColor = "danger"
+        } else {
+            bfOneClickColor = "success"
+        }
         var battleFront = document.getElementById('battleFrontierInformation').querySelector('div');
+        var oneClick = document.createElement("div");
```

Comment:
Avoid naming variables the same as functions you can access in the same scope, gets confusing

Reply:
Took your advice and renamed the var. Thanks!